demo/webpack.config.mjs

```diff
@@ -7,7 +7,11 @@ import HtmlWebpackPlugin from 'html-webpack-plugin'
 import ScriptExtHtmlWebpackPlugin from 'script-ext-html-webpack-plugin'
 import MiniCssExtractPlugin from 'mini-css-extract-plugin'
 import CopyWebpackPlugin from 'copy-webpack-plugin'
-import version from './tools/version'
+import version from './tools/version.mjs'
```

Comment:
I've tried to proceed using `webpack.config.babel.js` and `version.js`, but with no luck, thanks @NataliaDSmirnova for the help here